src/trio/_core/_tests/test_run.py

```diff
@@ -2505,9 +2505,12 @@ async def crasher() -> NoReturn:
 
     old_flags = gc.get_debug()
     try:
-        with RaisesGroup(
-            Matcher(ValueError, "^this is a crash$"),
-        ), _core.CancelScope() as outer:
+        with (
+            RaisesGroup(
+                Matcher(ValueError, "^this is a crash$"),
+            ),
+            _core.CancelScope() as outer,
+        ):
```

Comment:
This is a little wrong. It technically works, but parenthesising context managers is only officially supported in 3.10. It does work in 3.9, but not if you do `-X oldparser`. Though I guess black decided to rely on it anyway? But in this case it's rather more verbose.

Reply:
Fixed in 6499b69

Reply:
Made a PR upstream so eventually we can remove the ugly `fmt: off` and `fmt: on` things: https://github.com/psf/black/pull/4488